### PR TITLE
Fixes monkey cube exploit.

### LINKED ISSUE #44364: Ridiculous amount of gorillas spawned when cube dropped under the shower
Issue reported from  Round ID: 110025 (/tg/Station Sybil [ENGLISH] [US-WEST] [100% LAG FREE])

Reporting client version: 512.1464

This bug takes place when you drop the gorilla cube under the shower and take out the gorilla cubes individually and drop them while under a shower which is turned on
I don't really understand why it happens, but feel free to ask for more details if you need them

### PR DESCRIPTION
**Pull request by nemvar**
## About The Pull Request

Fixes an exploit that allows you to spawn an infinite amount of monkeys with a single cube.

Fixes #44364

## Why It's Good For The Game

No more infinite monky.

## Changelog
:cl:
fix: Fixes an exploit that allowed to you to spawn multiple monkeys with a single cube.
/:cl:

### CODE REVIEW

**Comment by spessbro** [comment]
gimme muh infinity monkeys

**Comment by Ty-the-Smonk** [comment]
I didnt even know this existed

**Comment by ma44** [comment]
The players have spoken

File: code/modules/food_and_drinks/food/snacks_meat.dm [hunk 1]
```diff
@@ -202,7 +202,7 @@
 		bananas.log_message("Spawned via [src] at [AREACOORD(src)], Last attached mob: [key_name(spammer)].", LOG_ATTACK)
 	else if (!spammer) // Visible message in case there are no fingerprints
 		visible_message("<span class='notice'>[src] fails to expand!</span>")
-	qdel(src)
+	QDEL_NULL(src)
```
Comment (AnturK) [inline code/modules/food_and_drinks/food/snacks_meat.dm]: I don't know what the issue here is but this won't be fixing it.

**Review by AnturK — CHANGES_REQUESTED** [review]

**Pull request closed without merging**